src/platform/user/profile/vet360/components/EmailField/VAPEmailField.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
```

Comment:
This file is based heavily on the existing `Vet360EmailField` but configures and renders a new `VAPProfileField` instead of the legacy `Vet360ProfileField`